manuscript/draft.Rmd

```diff
@@ -1,47 +1,58 @@
-# Title
+# Moving towards large-scale data-intensive forecasting of species richness in breeding birds
```

Comment:
How about something more like

*Best practices for forecasting biodiversity changes: an example with breeding birds*

I feel like my version does a better job of emphasizing that the paper is about best practices, not about breeding birds specifically. Also, I think "best practices" is less wishy-washy than "moving towards".  More like, "we know what to do, let's start doing it" and less like "I guess we should form a committee to figure out the way forward."